libs/kml/serdes_geojson.cpp

```diff
@@ -264,4 +285,132 @@ void DeserializerGeoJson::Deserialize(std::string_view content)
+
+    // Add '_umap_options' if needed.
+    if (auto const umapOptionsPair = bookmark.m_properties.find("_umap_options");
+        umapOptionsPair != bookmark.m_properties.end())
+    {
+      JsonTMap umap_options_obj;
+      if (auto error = glz::read_json(umap_options_obj, umapOptionsPair->second))
+      {
+        // Some error happened!
+        std::string err = glz::format_error(error, umapOptionsPair->second);
+        LOG(LWARNING, ("Error parsing '_umap_options' from KML properties:", err));
+      }
+      else
+      {
+        // Update known UMap properties.
+        umap_options_obj["color"] = ToCssColor(bookmark.m_color);
+        bookmarkProperties["_umap_options"] = umap_options_obj;
+      }
+    }
+
+    GeoJsonFeature pointFeature{.geometry = GeoJsonGeometryPoint{.coordinates = {lon, lat}},
+                                .properties = std::move(bookmarkProperties)};
+    geoJsonFeatures.push_back(std::move(pointFeature));
+  }
+
+  // Convert Tracks
+  for (size_t i = 0; i < fileData.m_tracksData.size(); i++)
+  {
+    TrackData const & track = fileData.m_tracksData[i];
+    auto linesCount = track.m_geometry.m_lines.size();
+    bool isMultiline = linesCount > 1;
+    if (linesCount == 0)
+      continue;
+    auto layer = track.m_layers[i];
+
+    JsonTMap trackProps{{"name", GetDefaultStr(track.m_name)}, {"stroke", ToCssColor(layer.m_color)}};
+    if (!track.m_description.empty())
+      trackProps["description"] = GetDefaultStr(track.m_description);
+
+    // Add '_umap_options' if needed.
+    if (auto const umapOptionsPair = track.m_properties.find("_umap_options");
+        umapOptionsPair != track.m_properties.end())
+    {
+      JsonTMap umap_options_obj;
+      if (auto error = glz::read_json(umap_options_obj, umapOptionsPair->second))
+      {
+        // Some error happened!
+        std::string err = glz::format_error(error, umapOptionsPair->second);
+        LOG(LWARNING, ("Error parsing '_umap_options' from KML properties:", err));
+      }
+      else
+      {
+        // Update known UMap properties.
+        umap_options_obj["color"] = ToCssColor(layer.m_color);
+        trackProps["_umap_options"] = umap_options_obj;
+      }
+    }
+
+    GeoJsonGeometry trackGeometry;
+    if (isMultiline)
+    {
+      std::vector<GeoJsonGeometryMultiLine::LineCoords> lines;
+      for (auto & trackLine : track.m_geometry.m_lines)
+        lines.push_back(convertPoints2GeoJsonCoords(trackLine));
+
+      trackGeometry = GeoJsonGeometryMultiLine{.coordinates = lines};
+    }
+    else
+    {
+      auto points = track.m_geometry.m_lines[0];
+      trackGeometry = GeoJsonGeometryLine{.coordinates = convertPoints2GeoJsonCoords(points)};
+    }
+    GeoJsonFeature trackFeature{.geometry = trackGeometry, .properties = trackProps};
+    geoJsonFeatures.push_back(std::move(trackFeature));
+  }
+
+  GeoJsonData geoJsonData{.features = geoJsonFeatures, .properties = std::nullopt};
```

Comment:
Always insert const if it compiles :)
```suggestion
  GeoJsonData const geoJsonData{.features = std::move(geoJsonFeatures), .properties = std::nullopt};
```